src/repo.go

```diff
@@ -311,7 +311,7 @@ func (repo *Repo) audit() error {
 					}
 					chunks := f.Chunks()
 					for _, chunk := range chunks {
-						if chunk.Type() == diffType.Add || chunk.Type() == diffType.Delete {
+						if (!opts.AdditionsOnly && chunk.Type() == diffType.Add) || chunk.Type() == diffType.Delete {
```

Comment:
@Zekii @zricethezav can we also propagate `chunk.Type()` value into json report?

That way, every leak will have additional property indicating where it came from - from addition or deletion.
Can be useful for analytics on top of standard gitleaks reports.

Reply:
Sorry about the late response, i'll check if it has been done by someone else already. But if not, i'll do it for sure, that's an nice data to have in reports ! :)

Reply:
Finally, after a month, i added the diffType to the report. Yaaay !

I noticed that the option `github-pr` doesn't only search in modified chunks but in the commit content, which contains the context (3 lines after and before the chunk). For example, if we take one of my commit:
```
 // Leak represents a leaked secret or regex match.
 type Leak struct {
        Line     string    `json:"line"`
+       DiffType string    `json:"diffType"`
        Commit   string    `json:"commit"`
        Offender string    `json:"offender"`
        Rule     string    `json:"rule"`
```

And a basic rule "Offender" would match in this case :(

First things first, i decided to not fix it since it is not the point of my PR.